andi/templates/ingress.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "helm.fullname" . }}
   namespace: {{ .Release.Namespace }}
   annotations:
+{{- if .Values.ingress.annotations }}
```

Comment:
I believe Ben and I are having issues with including these changes into our stuff, so to just articulate what's going on here. We're checking if there's local values and populating those and then if there's global's those overwrite those?

Reply:
The global values shouldn't overwrite these. It's to handle annotations that are specific to apps such as healthcheck urls.

Reply:
We just want to include both blocks of annotations if both are present.

Reply:
I think I understand. There are global annotations that are used by andi and other apps like you stated. But incase there are private andi ingress annotations include those too?

Reply:
Yes, that is accurate